bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/TeslaChannelSelectorProxy.java

```diff
@@ -160,7 +160,6 @@ public State getState(String s, TeslaChannelSelectorProxy proxy, Map<String, Str
                 return super.getState(s);
             }
         },
-        CAR_VERSION("car_version", "version", StringType.class, true),
```

Comment:
While software version might be a better naming convention then car version, but this also break existing (file based) setups, so what is the need to do this change?

Reply:
Actually, it does not as this was a property before. For the record, my complete setup is file based :)

Reply:
You are correct. Thought it was a channel, not a property. Still think it better fits as a property.